funnel/translations/hi_IN/LC_MESSAGES/messages.po

```diff
@@ -5759,12 +5758,12 @@ msgstr "ईवेंट का स्थान संपादित करे
 #: funnel/views/venue.py:142
 msgid "Delete venue “{title}”? This operation is permanent and cannot be undone."
 msgstr ""
-"\"{title}\" ईवेंट का स्थान मिटाएं? ऐसा करना इसे हमेशा के लिए मिटा देगा और"
-" इसे वापस पहले जैसा नहीं किया जा सकता."
+"“{title}” ईवेंट का स्थान मिटाएं? ऐसा करना इसे हमेशा के लिए मिटा देगा और "
+"इसे वापस पहले जैसा नहीं किया जा सकता."
```

Comment:
Not for this PR, but we need quote consistency: should be single quotes in all these places.